go/vt/vtctl/workflow/server.go

```diff
@@ -1075,10 +1149,15 @@ func (s *Server) getWorkflowCopyStates(ctx context.Context, tablet *topo.TabletI
 
 	copyStates := make([]*vtctldatapb.Workflow_Stream_CopyState, len(result.Rows))
 	for i, row := range result.Rows {
-		// These fields are technically varbinary, but this is close enough.
+		streamId, err := row[0].ToInt64()
+		if err != nil {
+			return nil, fmt.Errorf("failed to cast vrepl_id to int64: %v", err)
```

Comment:
IMO it's better to use vterrors in new code so that we can rely on having a meaningful code in more places over time. It's also helpful in these rare cases if we print the value we couldn't cast using `row[0].ToString()`. For example:
```
return nil, vterrors.Errorf(vtrpcpb.Code_INTERNAL, "failed to cast vrepl_id value of %q to int64: %v", row[0].ToString(), err)
```